source/Optimization.rst

```diff
@@ -0,0 +1,37 @@
+Optimisation
+===============
+
+
+PerTickCache
+----------------
+This function is used to retrieve values from the cache if some value is already computed. If the value is not present in the cache, it computes the value using a user-defined function and stores it in the cache.
```

Comment:
It isn't really a function, it's more a class in our example. But for now, you can just call it a "feature". Please go through the ``documentation.md`` file in the BharatSim repo that you have.